### PR TITLE
change files and finish format

### PR DESCRIPTION
**Pull request by petrukhinilya**

### CODE REVIEW

File: ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
-    const [showPopup, setShowPopup] = useState(false)
+  const [calendar, setCalendar] = useState([]);
+  const [value, setValue] = useState(moment());
+  const [showPopup, setShowPopup] = useState(false);
 
-    useEffect(() => {
-        setCalendar(builtCalendar(value))
-    }, [value])
+  useEffect(() => {
+    setCalendar(builtCalendar(value))
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js:15]: Missing semicolon at the end of line

File: ProjectCalendar/Frontend/src/Components/FormLogin.js [hunk 2]
```diff
@@ -14,10 +14,16 @@ const FormLogin = () => {
...
+    const {
+      target: {
+        children: {
+          password: {
+            value: password },
+          email: {
+            value: email },
+        }
+      }
+    } = e
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/FormLogin.js]: This destructuring aligned in a wrong way

File: ProjectCalendar/Frontend/src/Components/FormLogin.js [hunk 3]
```diff
@@ -14,10 +14,16 @@ const FormLogin = () => {
   const onLogin = (e) => {
     e.preventDefault()
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/FormLogin.js]: Missing semicolon at the end of line

File: ProjectCalendar/Frontend/src/Components/FormLogin.js (hunk 2 above)
Comment (d-lazarev) [reply ProjectCalendar/Frontend/src/Components/FormLogin.js]: Missing semicolon at the end of line

File: ProjectCalendar/Frontend/src/Components/FormRegister.js [hunk 4]
```diff
@@ -1,65 +1,74 @@
...
+                    value: password },
+                confirmPassword: {
+                    value: confirmPassword },
+                name: {
+                    value: name },
+                email: {
+                    value: email },
+                }
+            }
+        } = e;
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/FormRegister.js]: This destructuring aligned in a wrong way

File: ProjectCalendar/Frontend/src/Components/FormRegister.js [hunk 5]
```diff
@@ -1,65 +1,74 @@
...
+            setEmail(value);
+            break
+        case 'password':
+            setPassword(value);
+            break
+        case 'confirmPassword':
+            setConfirmPassword(value);
+            break
+        default:
+            break
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/FormRegister.js]: All `break` words should also end with a semicolon

File: ProjectCalendar/Frontend/src/Components/FormRegister.js [hunk 6]
```diff
@@ -1,65 +1,74 @@
...
+        <div>
+            <form onSubmit={onSubmit}>
+                <input onChange={onChange} type="text" placeholder="Name" name="name" value={name} />
+                <input onChange={onChange} type="text" placeholder="E-mail address" name="email" value={email} />
+                <input onChange={onChange} type="password" placeholder="Password" name="password" value={password} />
+                <input onChange={onChange} type="password" placeholder="Confirm password" name="confirmPassword" value={confirmPassword} />
+                <button type='submit'>Sign up </button>
+            </form>
+        </div>
+        )
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/FormRegister.js]: Should be align

File: ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js [hunk 7]
```diff
@@ -1,67 +1,67 @@
-import React, { useState, useEffect } from 'react'
-import moment from 'moment'
-import './Calendar.css'
-import builtCalendar from './BuiltCalendar'
+import React, { useState, useEffect } from 'react';
+import moment from 'moment';
+import './Calendar.css';
+import builtCalendar from './BuiltCalendar';
 import Popup from './Popup';
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js:5]: Sorting

File: ProjectCalendar/Frontend/src/Components/FormLogin.js [hunk 8]
```diff
@@ -1,8 +1,8 @@
-import React, { useState } from "react"
-import './Login.css'
-import paths from '../Routes/paths'
+import React, { useState } from "react";
+import './Login.css';
+import paths from '../Routes/paths';
 import { useHistory } from "react-router-dom";
-import { useDispatch } from 'react-redux'
+import { useDispatch } from 'react-redux';
 import { verifyUser } from '../Actions';
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/FormLogin.js:7]: Sorting

File: ProjectCalendar/Frontend/src/Components/FormRegister.js [hunk 9]
```diff
@@ -1,65 +1,74 @@
-import React ,{ useState } from 'react'
-import './Registration.css'
+import React, { useState } from 'react';
+import './Registration.css';
 import { useHistory } from "react-router-dom";
-import { useDispatch } from 'react-redux'
+import { useDispatch } from 'react-redux';
 import { addUser } from '../Actions';
-import paths from '../Routes/paths'
+import paths from '../Routes/paths';
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/FormRegister.js:7]: Sorting

**Review by d-lazarev — COMMENTED** [review]
Please see my notes below

File: ProjectCalendar/Frontend/src/Components/FormRegister.js (hunk 9 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/FormRegister.js:7]: Fixed

File: ProjectCalendar/Frontend/src/Components/FormRegister.js (hunk 5 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/FormRegister.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/FormLogin.js (hunk 8 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/FormLogin.js:7]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js (hunk 7 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js:5]: Fixed

File: ProjectCalendar/Frontend/src/Components/FormLogin.js (hunk 3 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/FormLogin.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/FormLogin.js (hunk 2 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/FormLogin.js]: Fixed both

File: ProjectCalendar/Frontend/src/Components/FormRegister.js (hunk 4 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/FormRegister.js]: fixed

File: ProjectCalendar/Frontend/src/Components/FormRegister.js (hunk 6 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/FormRegister.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js (hunk 1 above)
Comment (d-lazarev) [reply ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js:15]: Wasn't fixed

File: ProjectCalendar/Frontend/src/Components/FormRegister.js [hunk 10]
```diff
@@ -19,14 +22,18 @@ const FormRegister = () => {
...
+                    },
+                    confirmPassword: {
+                        value: confirmPassword 
+                    },
+                    name: {
+                        value: name 
+                    },
+                    email: {
+                        value: email 
+                    },
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/FormRegister.js]: Too much space from the start of the line, remove 1 Tabulation

**Review by d-lazarev — CHANGES_REQUESTED** [review]
See my notes below

File: ProjectCalendar/Frontend/src/Components/FormRegister.js (hunk 10 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/FormRegister.js]: Resolve tabulation

**Review by d-lazarev — APPROVED** [review]

**Pull request merged**